### PR TITLE
Fix toolBar.js relative path

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

**Comment by Atlante45** [comment]
Test plan:

recorder.js works

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5b1ba2f26c402fb041e3478f418e54fce116961f available at: https://gist.github.com/dac35d706d6bb9a9eb5d2dab51d33a23<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8573-5b1ba2f.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8573-5b1ba2f.dmg

File: scripts/developer/utilities/record/recorder.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
 //
 
 HIFI_PUBLIC_BUCKET = "http://s3.amazonaws.com/hifi-public/";
-Script.include("../../libraries/toolBars.js");
+Script.include("../../../system/libraries/toolBars.js");
```
Comment (sethalves) [inline scripts/developer/utilities/record/recorder.js]: this would also work:

```Script.include("/~/system/libraries/toolBars.js");```

**Comment by hifi-gustavo** [comment]
Build console output for commit: e7ebcc0467cce53e1caacfa0489d1892c8211e99 available at: https://gist.github.com/dac35d706d6bb9a9eb5d2dab51d33a23<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8573-e7ebcc0.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8573-e7ebcc0.dmg

**Comment by Nex-Pro** [comment]
I ran recorder.js and it appeared in the bottom left of my screen as expected.

**Even though recorder.js is not currently working I tried the various functions in case it offers some diagnostic value:**

It makes recordings that can be understood on playback ( if you knew what was said ). 

I recorded a short message. Clicking on stop moved my avatar to 0,0,0 in my sandbox.

I got this: Unhandled exception at 0x00007FFD49E89437 (ntdll.dll) in interface.exe: 0xC0000005: Access violation writing location 0x0000002CF8AE0F08. Whilst using the save and load buttons.

I also got thrown to a random location in my sandbox. 1727.3,920.4,832.0 after reloading interface and starting over.

While loop playback was active I was unable to move my avatar ( desktop mode ) other than rotating a couple of degrees in the vertical axis.

**Comment by Atlante45** [comment]
@Nex-Pro Thanks.

Yes, someone will have to revisit the feature. It appears some fairly "recent" work broke it.

**Pull request merged**